moveit/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="3">
   <name>moveit</name>
-  <version>2.1.1</version>
+  <version>2.1.2</version>
```

Comment:
and a version bump?

Reply:
I think it also included the changes in the main while rebasing I will have a look today.